custom/icds_reports/migrations/0081_auto_20181204_1213.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.16 on 2018-12-04 12:13
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('icds_reports', '0080_merge_20181130_1818'),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name='aggls',
+            old_name='unique_awc_vists',
+            new_name='awc_visits'
+        ),
+        migrations.RenameField(
+            model_name='aggregatelsawcvisitform',
+            old_name='unique_awc_vists',
+            new_name='awc_visits'
```

Comment:
Generally renaming columns is a bad practice as it would break any queries that are coming from old processes during deploy.

I think that its ok to rename them if there are no released reports which depend on these tables yet. Can you confirm that that is the case?